index.spec.js

```diff
@@ -0,0 +1,61 @@
+const request = require('supertest');
+
+const server = require('./api/server.js');
+
+describe('server', () => {
+    describe('GET /', () => {
+        
+        it('return status code 200(OK)', async () => {
+            const response = await request(server).get('/');
+
+            expect(response.status).toBe(200);
+        });
+    
+    })
+  
+    it('should return {message: "the line is hot"}', async () => {
+        const response = await request(server).get('/');
+
+        expect (response.body).toEqual('the line is hot');
+    });
+});
+
+
+describe('GET /api/games', () => {
+
+	it('return status 204 when no games are present', async () => {
+        const response = await request(server)
+            .get('/api/games');
+            
+        expect(response.status).toBe(204);
+        });
+
+    it('expected to return JSON', async () => {
+        const response = await request(server)
+            .get('/api/games');
+        expect(response.json).toBe([]);
```

Comment:
The expected result should be an array, not json. Use `isArray` to check for that.